src/Cake.Common/Tools/SignTool/SignToolSignSettings.cs

```diff
@@ -33,5 +33,10 @@ public sealed class SignToolSignSettings : ToolSettings
         /// Gets or sets the signed content's description.
         /// </summary>
         public string Description { get; set; }
+
+        /// <summary>
+        /// Gets or sets the signed content's expanded description URL.
+        /// </summary>
+        public Uri DescriptionUri { get; set; }
```

Comment:
I only went with `Uri` here (rather than `string`) to match what was already being done with `SignToolSignSettings.TimestampUri` (which is passed through as `/t` in the same manner).

I'm happy to change this to use string if folks think that's more appropriate.